dagstore.go

```diff
@@ -63,12 +69,13 @@ var (
 
 // DAGStore is the central object of the DAG store.
 type DAGStore struct {
-	lk      sync.RWMutex
-	mounts  *mount.Registry
-	shards  map[shard.Key]*Shard
-	config  Config
-	indices index.FullIndexRepo
-	store   ds.Datastore
+	lk            sync.RWMutex
+	mounts        *mount.Registry
+	shards        map[shard.Key]*Shard
+	config        Config
+	indices       index.FullIndexRepo
+	invertedIndex invertedindex.Index
```

Comment:
I think we should export this field, so that we can call the Index API from Lotus.

Reply:
@aarshkshah1992 can we export the `invertedIndex` field?

Reply:
Let's call this variable `topLevelIdx` (top level index).